File: DART/salario-mensal/SalarioMensal.dart
import 'dart:io';

void main() {
  double salary;
  int days;
  double hours;
  double total_hours;
  double per_hour;

  print('Insira o seu salário: ');
  salary = double.parse(stdin.readLineSync()!);
  salary = salary.toDouble();

  print('Insira os dias trabalhados: ');
  days = int.parse(stdin.readLineSync()!);
  days = days.toInt();

  print('Insira as horas trabalhadas: ');
  hours = double.parse(stdin.readLineSync()!);
  hours = hours.toDouble();

  total_hours = days * hours;
  per_hour = salary / total_hours;

  print("Você recebe por hora R\$" + per_hour.toStringAsPrecision(4));
}

File: JAVA/salario_mensal/SalarioMensal.java
package salario_mensal;

import java.text.NumberFormat;
import java.util.Scanner;

public class SalarioMensal {

    static NumberFormat numberFormat = NumberFormat.getCurrencyInstance();
    public static void main(String[] args) {

        
        Double salary;
        Integer days;
        Double hours;
        Double total_hours;
        Double per_hour;
        
        Scanner scanner = new Scanner(System.in);

        System.out.print("Insira o salário: ");
        salary = scanner.nextDouble();

        System.out.print("Insira os dias trabalhados: ");
        days = scanner.nextInt();

        System.out.print("Insira as horas trabalhadas: ");
        hours = scanner.nextDouble();

        total_hours = days * hours;
        per_hour = salary / total_hours;

        System.out.print("Você recebe por hora: R$"+numberFormat.format(per_hour));
    }
}